apps/nextjs-website/src/components/molecules/DownloadableDocuments/DownloadableDocuments.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+import React from 'react';
+import { Box, Grid, Typography } from '@mui/material';
+import Download from '@mui/icons-material/Download';
+import CtaCard from '@/components/atoms/CtaCard/CtaCard';
+import { useTranslations } from 'next-intl';
+
+const getReadableFileSizeString = (fileSizeInBytes: number) => {
+  // eslint-disable-next-line functional/no-let
+  let i = -1;
+  // eslint-disable-next-line functional/no-let
+  let size = fileSizeInBytes;
+  const byteUnits = [' kB', ' MB', ' GB', ' TB', 'PB', 'EB', 'ZB', 'YB'];
+  // eslint-disable-next-line functional/no-loop-statements
+  do {
+    size /= 1024;
+    i++;
+  } while (size > 1024);
+
+  return Math.max(size, 0.1).toFixed(1) + byteUnits[i];
+};
+
+export type DownloadableDocumentsProps = {
+  documents: {
+    title: string;
+    tags?: { readonly label: string; readonly path?: string }[];
+    downloadLink?: string;
+    size?: number;
+  }[];
+};
+
+const DownloadableDocuments = ({ documents }: DownloadableDocumentsProps) => {
+  const t = useTranslations('shared');
```

Comment:
I know that somewhere in this project there are old nested translation but we have defined to avoid passing argument to `useTranslations` function and instead use `shared.NAME` 
```suggestion
  const t = useTranslations();
```